chaos-monkey-docs/src/main/asciidoc/changes.adoc

```diff
@@ -1,6 +1,6 @@
 [[changes]]
 == Changes in {project-version}
-
+https://github.com/codecentric/chaos-monkey-spring-boot/pull/131 Deprecated constructor usage removal
```

Comment:
Please us a markdown link here like: `- https://github.com/codecentric/chaos-monkey-spring-boot/pull/131[#131] Update dependencies to latest stable versions.` Please note, that this is a list!